airbyte-integrations/connectors/destination-duckdb/destination_duckdb/spec.json

```diff
@@ -0,0 +1,21 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/duckdb",
+  "supported_destination_sync_modes": [ "overwrite" ],
+  "supportsIncremental": false,
+  "supportsDBT": false,
+  "supportsNormalization": false,
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Destination Duckdb",
+    "type": "object",
+    "required": ["destination_path"],
+    "additionalProperties": false,
+    "properties": {
+      "destination_path": {
+        "type": "string",
+        "description": "Path to the destination.ducdb file. The file will be placed inside that local mount. For more information check out our <a href=\"https://docs.airbyte.io/integrations/destinations/duckdb\">docs</a>",
+        "example": "/local/destination.duckdb"
+      }
```

Comment:
Why you don't connect to a duckdb instance/docker instead of saving it locally?

Reply:
worked after resetting everything